docs/releases/1.10.txt

```diff
@@ -732,6 +732,17 @@ use the default_related_name ``bars``::
 
     >>> Foo.object.get(bars=bar)
 
+Custom related manager must define a ``_apply_rel_filters()`` method
+--------------------------------------------------------------------
+
+If you defined a custom manager class available through
+:meth:`~django.db.models.query.QuerySet.prefetch_related` you must make sure
```

Comment:
I don't think an example is required, I believe this will affect a really small subset of users.

I'll make sure to create PR on django-taggit's project and search for other project defining custom manager classes.